Makefile

```diff
@@ -1,4 +1,4 @@
-VER ?= 5.2.0
+VER ?= 6.2.0
```

Comment:
@benarent can you check latest available version for mattermost and gravity again? After bumping them we are ready to merge.
LGTM from me.